Vagrantfile

```diff
@@ -10,7 +10,7 @@ cd /vagrant
 sudo ./bootstrap/ubuntu.sh
 if [ ! -d "venv" ]; then
   virtualenv --no-site-packages -p python2 venv
-  ./venv/bin/pip install -r requirements.txt -e .[dev,docs,testing]
+  ./venv/bin/pip install -e .
```

Comment:
I'd prefer to make sure that `git` is installed, or otherwise have a giant warning inline comment:
```python
# requirements.txt, at the time of writing, is necessary only for Python.2.6
# compatible ConfigArgParse and since Vagrant environment runs with
# Python2.7 this is omitted as it additionally requires git to be installed.
```

and there also should be a comment inside `requriements.txt`: `Vagrant doesn't use this file, c.f. Vagrantfile`. Note, that in (not so distant) future `requrements.txt` might be employed for other packages.

Vagrant is meant to be developer environment AFAIR. Therefore you also want to install `[dev,docs,testing]` extras.  In general, whenever you do `pip install -e .` you rather want to do `pip install -e .[dev,docs,testing]`

Every time I change the build setup even slightly I need to make sure all services are functional, hence it's better to keep them all setup similarily, or at least have good explanatory comments.  Thanks!